fastchat/model/model_adapter.py

```diff
@@ -98,6 +99,10 @@ def load_model(
     max_gpu_memory: Optional[str] = None,
     load_8bit: bool = False,
     cpu_offloading: bool = False,
+    load_gptq: str = "",
+    wbits: int = 16,
+    groupsize: int = -1,
+    act_order: bool = True,
```

Comment:
There are too many arguments. Can we try these two options to organize them better? Which one do you prefer?

1. rename, `wbits` -> `gptq_wbits`, `groupsize` -> `gptq_group_sizze`, `act_order` -> `gotq_act_order`.
2. Merge `load_gptq`, `wbits`, `groupsize`, `act_order` into a single argument, which can be a tuple/dict or a new class called `GptqConfig`

Reply:
Thanks for your valuable suggestions, I will try it first to see which one is better.